test/functional/fixtures/reporter/testcafe-fixtures/report-data-test.js

```diff
@@ -0,0 +1,16 @@
+fixture`Report Data API`
+    .page('../pages/index.html');
+
+test('Run t.report action twice', async t => {
```

Comment:
Actually, these tests are the same.
Leave the only one test with the following content:
```js
test('Run t.report action with object val', async t => {
    await t
        .report('Report 1')
        .report(1, true, 'string', { 'reportResult': 'test' });
});
```